packages/server/migrations/1596574321607-migration.ts

```diff
@@ -0,0 +1,34 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class migration1596574321607 implements MigrationInterface {
+    name = 'migration1596574321607'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+      await queryRunner.query('CREATE TABLE "flower" ("id" SERIAL NOT NULL, "shelfId" integer NOT NULL, "name" character varying NOT NULL, "description" character varying NOT NULL, "pictureUrls" text NOT NULL, "rrules" text NOT NULL, CONSTRAINT "PK_8d89826a15a9d5d16567c055637" PRIMARY KEY ("id"))');
+      await queryRunner.query('CREATE TABLE "shelf" ("id" SERIAL NOT NULL, "name" character varying NOT NULL, "description" character varying NOT NULL, "location" character varying NOT NULL, "pictureUrl" character varying NOT NULL, CONSTRAINT "PK_da2ce57e38dfc635d50d0e5fc8f" PRIMARY KEY ("id"))');
+      await queryRunner.query('CREATE TABLE "shelf_flower_ordering" ("shelfId" integer NOT NULL, "flowerIds" text NOT NULL, CONSTRAINT "PK_5bf93b449034bf086c66a8d50c5" PRIMARY KEY ("shelfId"))');
+      await queryRunner.query('CREATE TYPE "user_shelf_duties_role_enum" AS ENUM(\'admin\', \'keeper\')');
+      await queryRunner.query('CREATE TYPE "user_shelf_duties_actions_enum" AS ENUM(\'watering\', \'hydration\', \'fertilizing\')');
+      await queryRunner.query('CREATE TABLE "user_shelf_duties" ("shelfId" integer NOT NULL, "role" "user_shelf_duties_role_enum" NOT NULL, "actions" "user_shelf_duties_actions_enum" array NOT NULL, "userId" character varying NOT NULL, CONSTRAINT "PK_5da14309f1f99fc7f76aef46c9c" PRIMARY KEY ("shelfId", "userId"))');
+      await queryRunner.query('CREATE TYPE "user_shelf_invitation_role_enum" AS ENUM(\'admin\', \'keeper\')');
+      await queryRunner.query('CREATE TYPE "user_shelf_invitation_actions_enum" AS ENUM(\'watering\', \'hydration\', \'fertilizing\')');
+      await queryRunner.query('CREATE TABLE "user_shelf_invitation" ("shelfId" integer NOT NULL, "role" "user_shelf_invitation_role_enum" NOT NULL, "actions" "user_shelf_invitation_actions_enum" array NOT NULL, "token" character varying NOT NULL, "userEmail" character varying NOT NULL, "expiresIn" bigint NOT NULL, CONSTRAINT "PK_606582f4960ccb362e04fa76454" PRIMARY KEY ("shelfId", "token"))');
+      await queryRunner.query('CREATE TABLE "user_shelf_ordering" ("userId" integer NOT NULL, "shelfIds" text NOT NULL, CONSTRAINT "PK_a434854aa3204a945eaabd9349a" PRIMARY KEY ("userId"))');
+      await queryRunner.query('ALTER TABLE "user" DROP COLUMN "googleId"');
```

Comment:
Should this change be done in current PR?

Reply:
It was just a leftover from the last migration Tolik did. Could be a separate fix, but it never happened, so it got here.

Reply:
Would you prefer a separate fix PR for this?

Reply:
It would be great.

Reply:
OK, but I would have to regenerate this migration based on that one, so let's merge the intermediate PR fast then.

Reply:
Hm.. Seems that I can't create an intermediate migration without first changing something in the entities:
```
$ yarn migration:generate:windows
yarn run v1.21.1
$ ts-node ../../node_modules/typeorm/cli.js migration:generate -n migration
Error during migration generation:
TypeError: Cannot read property 'map' of undefined
    at PostgresQueryRunner.createEnumTypeSql (C:\d\git\dubbha\croton\src\driver\postgres\PostgresQueryRunner.ts:1845:41)
    at PostgresQueryRunner.<anonymous> (C:\d\git\dubbha\croton\src\driver\postgres\PostgresQueryRunner.ts:505:37)
    at step (C:\d\git\dubbha\croton\node_modules\tslib\tslib.js:141:27)
    at Object.next (C:\d\git\dubbha\croton\node_modules\tslib\tslib.js:122:57)
    at fulfilled (C:\d\git\dubbha\croton\node_modules\tslib\tslib.js:112:62)
    at processTicksAndRejections (internal/process/task_queues.js:94:5)
error Command failed with exit code 1.
```
Not sure how the previous migration was created, probably some manual changes in DB involved.

Reply:
Let's maybe keep it here?

Reply:
Fixed, [separate PR](https://github.com/dubbha/croton/pull/83) merged for the google ID removal, [enum fix](https://github.com/dubbha/croton/pull/81/files#diff-f040aa1fdc8693258a511b3bd5f48cceR31) here fixes the `createEnumTypeSql` error I had above